medicines/web/src/services/keyword.ts

```diff
@@ -0,0 +1,38 @@
+const searchExactnessBoost = process.env.AZURE_SEARCH_EXACTNESS_BOOST;
```

Comment:
This file name could be mistaken for the use of `keywords` relating to the field of the same name. I think it'd be better renamed to e.g. `search-query-normalizer` or `search-query-cleaner`.

Reply:
Renamed